pandas/_version.py

```diff
@@ -205,10 +205,7 @@ def git_pieces_from_vcs(tag_prefix, root, verbose, run_command=run_command):
         raise NotThisMethod("'git rev-parse' failed")
     full_out = full_out.strip()
 
-    pieces = {}
-    pieces["long"] = full_out
-    pieces["short"] = full_out[:7]  # maybe improved later
-    pieces["error"] = None
+    pieces = {"long": full_out, "short": full_out[:7], "error": None}
```

Comment:
I've done:
git fetch upstream (with upstream being github:pydata/pandas)
git rebase upstream/master
git push -u origin -f dict_litteral:dict_litteral

Plus I've checked my code against the master and it maches. Would it help
if I submitted the modification on the _version.py in a separate pull
request?
If that's too much hassle I can close this PR...

2015-11-14 16:24 GMT+01:00 Jeff Reback <notifications@github.com>:

> In pandas/_version.py
> <https://github.com/pydata/pandas/pull/11538#discussion_r44859699>:
>
> > @@ -205,10 +205,7 @@ def git_pieces_from_vcs(tag_prefix, root, verbose, run_command=run_command):
> >          raise NotThisMethod("'git rev-parse' failed")
> >      full_out = full_out.strip()
> >
> > -    pieces = {}
> > -    pieces["long"] = full_out
> > -    pieces["short"] = full_out[:7]  # maybe improved later
> > -    pieces["error"] = None
> > +    pieces = {"long": full_out, "short": full_out[:7], "error": None}
>
> doesn't appear to be the case. you still have the original commits.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pydata/pandas/pull/11538/files#r44859699>.
>